deck-core/src/main/kotlin/com/deck/core/cache/DeckCacheManager.kt

```diff
@@ -36,4 +39,12 @@ public class DeckCacheManager() : CacheManager {
     override fun retrieveTeam(id: GenericId): Team? {
         return teams.getIfPresent(id)
     }
+
+    override fun retrieveMessage(id: UniqueId): Message? {
+        return messages.getIfPresent(id)
+    }
+
+    override fun updateMessage(id: UniqueId, message: Message) {
```

Comment:
This should be nullable, in case you want to delete a message from the cache

```suggestion
    override fun updateMessage(id: UniqueId, message: Message?) {
```